### DISCUSSION TITLE
guides/operations/kubernetes/charts/lab/

### DISCUSSION THREAD

**Discussion by giscus** [odpi/egeria-docs#879] [DocFeedback]
# guides/operations/kubernetes/charts/lab/

Open metadata and governance for enterprises - automatically capturing, managing and exchanging metadata between tools and platforms

https://egeria-project.org/guides/operations/kubernetes/charts/lab/

**Comment by ksambhav** [comment]
It would help if we can add some screenshots here. Great documentation overall.
